apps/omg_watcher/lib/omg_watcher/application.ex

```diff
@@ -14,21 +14,16 @@
 
 defmodule OMG.Watcher.Application do
   @moduledoc false
-  alias OMG.Watcher.Alert.AlarmHandler
-  require Logger
   use Application
-  use OMG.LoggerExt
+  require Logger
```

Comment:
this change is done in a couple of places - I'm curious why?

Reply:
I think wrapping the logger in a macro would prevent us to purge the code at compile time for releases.

Reply:
ah, `LoggerExt` dates to from times before this purging was released.

But maybe let's opt towards fixing that in a separate PR? there are lots of `use OMG.LoggerExt` left still. And also `LoggerExt` is (or at least used to be) used by some of us. Maybe it's worth to discuss alternatives or fix the mechanics of `LoggerExt` to have the functionality it offered.

Reply:
By removing the custom logger reference from rpc I’m able to completelly separate rpc from omg which is super beneficial. I can readd the logger but it would be implemented in Rpc  otherwise we could move the logger to utils.